cvat/apps/dataset_manager/formats/transformations.py

```diff
@@ -32,3 +35,16 @@ def transform_item(self, item):
                 z_order=ann.z_order))
 
         return item.wrap(annotations=annotations)
+
+class EllipsesToMasks(ItemTransform):
+    @staticmethod
+    def convert_ellipse(ellipse, img_h, img_w):
+        cx, cy, rightX, topY = ellipse.points
+        rx, ry = rightX - cx, cy - topY
```

Comment:
```suggestion
        rx = rightX - cx
        ry = cy - topY
```
I don't recommend using multiple expressions in a line, because it makes reading worse. A matter of taste, though.